package.json

```diff
@@ -31,7 +31,8 @@
     "lint": "yarn lint:eslint && yarn lint:prettier",
     "lint:eslint": "eslint --ext .ts,.tsx . -f unix",
     "lint:fix": "eslint --ext .ts,.tsx . -f unix --fix && prettier --write **/*.{ts,tsx} *.js",
-    "lint:prettier": "prettier --check **/*.{ts,tsx} *.js"
+    "lint:prettier": "prettier --check **/*.{ts,tsx} *.js",
+    "version": "lerna bootstrap"
```

Comment:
This is a workaround to [this bug](https://github.com/lerna/lerna/issues/1171) in Lerna.

Reply:
You can also read more about [lifecycle scripts in `lerna version`](https://github.com/lerna/lerna/tree/master/commands/version#lifecycle-scripts)